linux/guest_customization/check_gosc_support_status.yml

```diff
@@ -19,6 +19,16 @@
     skip_reason: "Blocked"
   when: enable_cloudinit_gosc and not vmtools_is_ovt
 
+- name: "Skip test case"
+  include_tasks: ../../common/skip_test_case.yml
+  vars:
+    skip_msg: "Test case '{{ ansible_play_name }}' is skipped because it's not supported"
```

Comment:
Please update the `skip_msg` to make it more clear, like:
```
Test case '{{ ansible_play_name }}' is skipped because guest id {{ vm_guest_id }} doesn't support guest customization on vCenter Server {{ vcenter_version }}
```